code/game/atoms.dm

```diff
@@ -30,11 +30,11 @@
 
 	var/gfi_layer_rotation = GFI_ROTATION_DEFAULT
 
-	// Extra descriptions.
-	var/desc_fluff = null // Text about the atom's fluff description, if any exists.
+	// Extra descriptions. // Regular descriptions typically reserved for dry, raw descriptions.
+	var/desc_fluff = null // Text about the atom's fluff description, if any exists. Reserved for funny jokes and snarky comments, mostly.
```

Comment:
This is not how it has been used in a lot of items, that use serious lore description.

Reply:
The problem I'm trying to fix is that desc_fluff is both used for serious lore description and jokes.
With this PR I'm planning to seperate them out, so serious lore uses desc_lore and jokes use desc_fluff.